core/blockchain.go

```diff
@@ -1521,7 +1521,6 @@ func (bc *BlockChain) InsertHeaderChain(chain []*types.Header, checkFreq int) (i
 	if i, err := bc.hc.ValidateHeaderChain(chain, checkFreq); err != nil {
 		return i, err
 	}
-
```

Comment:
?

Reply:
I made several debug info in the code, not cleaned up fully

Reply:
add space back